src/schema/rules/datatypes/derivatives/common_derivatives.yaml

```diff
@@ -0,0 +1,413 @@
+---
+anat_nonparametric_common:
+  $ref: schema.rules.datatypes.anat.nonparametric
```

Comment:
Quick FYI as I browsing the several schema related PRs

starting those keys with a `$` will for sure not play nice with MATLAB in the end, unless I change them to something else when converting from yaml to json in the bids-matlab repo but I would prefer to avoid having different flavor of the schema.

Reply:
`_ref` would presumably work in all languages and be unlikely to collide with valid keys anywhere we might use this construct.